Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/ScaleManipulators.h

```diff
@@ -44,12 +44,12 @@ namespace AzToolsFramework
         //! Sets the bound width to use for the line/axis of a linear manipulator.
         void SetLineBoundWidth(float lineBoundWidth);
 
+        // Manipulators
```

Comment:
nit: Please can we remove this comment, it's redundant here and adds nothing for the reader